### PR TITLE
POCL Test File Generator

### PR DESCRIPTION
**Pull request by jaucourt**
https://eaflood.atlassian.net/browse/IWTF-1074

Add a POCL Test File Generator to:
- produce a configurable number of records

### CODE REVIEW

File: packages/pocl-test-file-generator/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import poclTestFileGenerator from 'commander'
```
Comment (graham-willis-druid) [inline packages/pocl-test-file-generator/index.js:1]: Just a thought - would this be a good place to try out a lambda function (triggered via the gateway)  not suggesting doing it now, but rather a candidate for a future PoC

File: packages/pocl-test-file-generator/src/generateRecord.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+})
+
+const getRandomDateOfBirth = ageRange => {
+  const minAge = minAndMaxAgesForRange[ageRange].min
+  const maxAge = minAndMaxAgesForRange[ageRange].max
+  return moment().subtract(randomInteger(minAge, maxAge), 'years').subtract(1, 'day')  
+}
+
+const minAndMaxAgesForRange = Object.freeze({
+  [AGE_RANGE.JUNIOR]: Object.freeze({ min: JUNIOR_MIN_AGE, max: JUNIOR_MAX_AGE }),
```
Comment (graham-willis-druid) [inline packages/pocl-test-file-generator/src/generateRecord.js:26]: Another thought - I think freeze only works in strict mode, and we get strict mode implicitly by using the nodule system. have you actually tried it?

**Review by graham-willis-druid — APPROVED** [review]
Looks good

File: packages/pocl-test-file-generator/src/generateRecord.js (hunk 2 above)
Comment (jaucourt) [reply packages/pocl-test-file-generator/src/generateRecord.js:26]: Just tried this in `generateRecord.test.js`:

  it('freezes value of AGE_RANGE', () => {
    expect(() => {
      AGE_RANGE.ADULT = 4
    }).to.throw()
  });

and it passes (and fails when we're not using Object.freeze)

File: packages/pocl-test-file-generator/index.js (hunk 1 above)
Comment (jaucourt) [reply packages/pocl-test-file-generator/index.js:1]: Yeah, this package would probably be a decent candidate for a lambda function

**Pull request merged**